yast-travis-ruby

```diff
@@ -181,6 +196,15 @@ function run_rubocop() {
   fi
 }
 
+# check the syntax of the perl files
+function check_perl() {
+  # Perl allows checking the syntax only for one file at once, we need to use -n1
+  # xargs option, to speed it up run the checks in parallel (-P option).
+  # If you need to specify an additional search path use the PERL5LIB environment variable.
+  find . \( -name '*.pm' -o -name '*.pl' \) -print0 \
```

Comment:
IMHO this is more readable
`find . -name '\*.p[lm]' -print0`
(also replace in RUN_CHECK_PERL init above)